content/en/docs/setup/platform-setup/_index.md

```diff
@@ -9,4 +9,4 @@ keywords: [platform-setup]
 
 {{< tip >}}
 Istio {{< istio_version >}} has been tested with these Kubernetes releases: {{< supported_kubernetes_versions >}}.
-{{< /tip >}}
+{{</ tip >}}
```

Comment:
The correct syntax, AFAIK, is:
```suggestion
{{< /tip >}}
```
Did your syntax render correctly?

Reply:
You're right. It has no problem rendering because there was nothing after this text. Even if you leave the ending tag, it's still okay. That's why I didn't see any issue. Will make the change. Thanks!